src/components/weather-alerts/weather-alert-banner.tsx

```
@@ -0,0 +1,59 @@
+interface WeatherAlertBannerProps {
+  alerts: WeatherAlertResultData[];
+  onPress: () => void;
+  onDismiss: () => void;
+}
+
+export const WeatherAlertBanner: React.FC<WeatherAlertBannerProps> = ({ alerts, onPress, onDismiss }) => {
+  if (alerts.length === 0) {
+    return null;
+  }
+
+  const { t } = useTranslation();
```

ESLint

[error] React Hook "useTranslation" is called conditionally. React Hooks must be called in the exact same order in every component render.

(react-hooks/rules-of-hooks)